docs/getting-started/prerequisites.md

```diff
@@ -10,4 +10,5 @@ In order to run you need to have installed:
 - [Docker](https://www.docker.com/)
 - [Docker Compose](https://docs.docker.com/compose/)
 - Git
-- [Python](https://www.python.org/) (3.10 or newer)
\ No newline at end of file
+- [Python](https://www.python.org/) (3.10 or newer)
+- [NodeJS](https://nodejs.org/en/)
```

Comment:
Could put "python" and "node" under a separate "If you want to run locally".
Could then also add "poetry" and "yarn"